deployments/ynETHx-1-v0.0.1.json

```diff
@@ -50,7 +47,6 @@
       "isL1": false,
       "lzEID": 30255,
       "lzEndpoint": "0x1a44076050125825900e736c501f859c50fE728c",
-      "multiChainDeployer": "0xDE21c853BA39e77251a90dBB0b4C3d94463EbCfA",
```

Comment:
multiChainDeployer address gets removed automatically since the whole deployment json is rewritten by the deploy script